packages/binding-http/src/http-server.ts

```diff
@@ -752,7 +775,7 @@ export default class HttpServer implements ProtocolServer {
                                                     return;
                                                 }
                                                 // send event data
-                                                res.end(content.body);
+                                                content.body.pipe(res);
```

Comment:
while we are here I think this can be moved inside the try block.

Reply:
👍